app/assets/stylesheets/left_nav.scss

```diff
@@ -5,9 +5,20 @@ $beige: #D4C99E;
   background-color: $beige;
 }
 
+#left_nav_show {
```

Comment:
Avoid using id selectors<br>Selector `left_nav_show` should be written in lowercase with hyphens